plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -45,14 +45,25 @@ static inline int key_cmp(const char *str, int len, const char *cmp) {
 
 static int validate_resource(const char *res)
 {
-    if (strcasecmp(res, "global") != 0 &&
-        strcasecmp(res, "gce_instance") != 0 &&
-        strcasecmp(res, "generic_node") != 0 &&
-        strcasecmp(res, "generic_task") != 0) {
-        return -1;
+    /*
+        * Resource types
+        * 'global', 'gce_instance', `generic_node', 'generic_task' are supported
+    */
+    char *valid_resources[] = {
+        "global",
+        "gce_instance",
+        "generic_node",
+        "generic_task",
+        NULL
+    };
+
+    for(char **r = valid_resources; *r; ++r) {
```

Comment:
I'll let @edsiper comment on style compliance, but I think fluent bit adds spaces after `for` and `if`.

Reply:
They do, will fix.

Reply:
Done